backend/app/src/main/java/com/moh/phlat/backend/service/ProcessDataService.java

```diff
@@ -1,14 +1,16 @@
 package com.moh.phlat.backend.service;
 
+import java.util.Arrays;
+import java.util.List;
+
 import com.moh.phlat.backend.model.ProcessData;
 import com.moh.phlat.backend.model.ProcessDataFilterParams;
 import com.moh.phlat.backend.service.dto.ReportSummary;
 
-import java.util.Arrays;
-import java.util.List;
-
 public interface ProcessDataService {
-	
+	String PHLAT_ERROR_TYPE = "ERROR";
+    String PHLAT_ERROR_CODE = "100";
```

Comment:
```suggestion
    String PHLAT_ERROR_CODE = "PHLAT.100";
```
This is to avoid hard coding "PHLAT" in the messages.properties